### DISCUSSION TITLE
featurerequest: support for Windows on ARM64

### DISCUSSION THREAD

**Discussion by zedd-0** [doublecmd/doublecmd#1924] [Suggestions]
add support for windows on arm(64).
i need it for PE-systems, without emulations-layer.
thx.

**Comment by alexx2000** [comment]
Free Pascal/Lazarus does not support this platform yet.

**Comment by alexx2000** [comment]
Free Pascal 3.4 will be support it.

**Comment by cristianadam** [comment]
I would like to have a native arm64 version. Here is the corresponding Total Commander [forum](https://www.ghisler.ch/board/viewtopic.php?p=426630) post.

An arm64 laptop is nice for not being noisy, not getting very hot under 100% usage and having a long battery life.

**Comment by cristianadam** [comment]
The corresponding Free Pascal [Free Pascal for Windows AARCH64 (ARM64)](https://forum.lazarus.freepascal.org/index.php?topic=48899.0) post. Not very promising though.

Given the fact that you can have Windows 11 Arm64 on a Raspberry PI4 as seen at https://raspberrytips.com/windows-11-on-raspberry-pi/ you can't really say that's not enough hardware out there.

**Comment by alexx2000** [comment]
Free Pascal 3.3.1 alpha already supports aarch64-win64 target. But it's too buggy yet.

**Comment by lexalex1** [comment]
I'm waiting for doublecmd for windows-arm too...

**Comment by alexx2000** [comment]
https://forum.lazarus.freepascal.org/index.php/topic,66952.75.html
